modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APISchemaRelevantObjectEntryModelListenerTest.java

```diff
@@ -80,6 +80,22 @@ public void test() throws Exception {
 			"An API schema must be an existing object definition.",
 			jsonObject.get("title"));
 
+		jsonObject = HTTPTestUtil.invokeToJSONObject(
+			JSONUtil.put(
+				"mainObjectDefinitionERC", "L_USER"
+			).put(
+				"name", RandomTestUtil.randomString()
+			).put(
+				"r_apiApplicationToAPISchemas_c_apiApplicationId",
+				apiApplicationJSONObject.getLong("id")
+			).toString(),
+			"headless-builder/schemas", Http.Method.POST);
+
+		Assert.assertEquals("BAD_REQUEST", jsonObject.get("status"));
+		Assert.assertEquals(
+			"An API schema must be a modifiable object definition.",
```

Comment:
I think this sub-test should be below the existing as the message is not properly sorted:

`An API schema must be related to an API application.` > `An API schema must be a modifiable object definition.`